### PR TITLE
<TBBAS-1390> Input status of scaling function block

### PR DESCRIPTION
**Pull request by NikolaiShipilov**
## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] Pull request title reflects its content
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Description:
Adds an implementation example for component statuses - input status of scaling function block

### CODE REVIEW

File: modules/ref_fb_module/src/scaling_fb_impl.cpp [hunk 1]
```diff
@@ -239,6 +255,36 @@ void ScalingFbImpl::createSignals()
     outputSignal.setDomainSignal(outputDomainSignal);
 }
 
+void ScalingFbImpl::initStatuses()
+{
+    auto inputStatusType = EnumerationType("InputStatusType", List<IString>(InputDisconnected, InputConnected, InputInvalid));
+    this->context.getTypeManager().addType(inputStatusType);
```
Comment (JakaMohorkoDS) [inline modules/ref_fb_module/src/scaling_fb_impl.cpp:261]: This should probably check for whether or not it exists already.

**Review by JakaMohorkoDS — APPROVED** [review]
Minor comment about adding types to the type manager - duplicates are not allowed.

File: modules/ref_fb_module/src/scaling_fb_impl.cpp (hunk 1 above)
Comment (NikolaiShipilov) [reply modules/ref_fb_module/src/scaling_fb_impl.cpp]: Fixed

**Comment by JakaMohorkoDS** [comment]
Lgtm now.

**Pull request merged**